README.md

```diff
@@ -128,15 +128,15 @@ You can deploy the app to `Vercel` or `Netlify` with a single click.
 </a>
 
 ## 🤝 Contributing to `ReactPlay`
-Any kind of positive contributions are welcome! Please help us to grow by contributing to the project.
+Any kind of positive contribution is welcome! Please help us to grow by contributing to the project.
```

Comment:
@hasnainmakada-99, I have changed the `Any kind of positive contributions are welcome` -- > `Any kind of positive contribution is welcome`

Reply:
@hasnainmakada-99 , Let me know if you were asking for something else.

Reply:
@hasnainmakada-99 , You can change the diff view to `split` and see what things have changed.

<img width="1411" alt="image" src="https://user-images.githubusercontent.com/59080942/172536239-8c6e0b28-e3ca-4eea-a4c1-a024e2285425.png">

Reply:
@Sachin-chaurasiya @hasnainmakada-99 Is it open for review? Just asking so that doesn't wait for the merge.

Reply:
@atapas , From my end all changes are done, waiting for a @hasnainmakada-99 reply.

Reply:
@Sachin-chaurasiya  I was telling you to that change the text to `Any kind of contributions are welcome ` from `Any kind of contributin is welcome` because contributions can be more than one from a single user, Are you getting me

Reply:
@hasnainmakada-99 , but technically `Any kind of positive contribution is welcome! Please help us to grow by contributing to the project.` this is the correct statement.


cc : @atapas.

Reply:
Ya that also sounds right if everyone thinks this is the all the reviewers have finalized this statement then its good to go, I was just telling from my side @Sachin-chaurasiya

Reply:
I am resolving this conversation now